client/src/features/bulletins/host/BulletinForm/BulletinForm.tsx

```diff
@@ -23,54 +21,46 @@ enum sections {
 }
 
 type Props = {
-  bulletin?: Bulletin;
-  onCancel?: () => void;
+  bulletin: Bulletin;
+  onSubmit: (bulletinBody: BulletinBody, id: string) => void;
```

Comment:
Come fa a sapere l'id il `BulletinForm`? Toglilo dalla firma, in creazione non è richiesto e in modifica lo sa il componente padre già